### ISSUE TITLE
Xenoarch consistency issues

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#21154]
(WEB REPORT BY: shadownil REMOTE: 206.221.180.138:7777)
# Description
Fourier Transform Spectroscope in Xenoarch Lab is supposed to give you a distance value or something, but all it gives is accuracy that the large artifact is in "emission radius" and then a number. Wiki doesn't explain this well at all, and there's no way I can tell to obtain an answer like "the large artifact is THIS many tiles away from that spot". What are you supposed to do to the number to determine how many tiles away the large artifact is from the rock wall you got the sample from? Things like that.
On the other hand, Hyperspectral Imager does tell you which artifact got pinged by the sample, but there's a broken icon (white box with black box with a white X in it) under "Spectral signature over carrier (???):", which is the same sort of broken icon that shows up in some of the tile painter options.
Not to mention, this pic (http://ss13.moe/wiki/images/4/4e/Artifact_Discovery.png) on the Guide to Xenoarch page posted by Clusterfack doesn't help convey or explain exactly how the fuck triangulation works. The wiki page is also very unclear when it comes to explaining what the results of the xenoarch machines are supposed to be telling the player.

Bottom line: Xenoarch needs to be explained a bit clearer on the wiki and in the game itself.

**Comment by ghost** [comment]
It's fine dude, just read. 
Seriously, it's bulleted and everything. It told you emission radius is distance/2 = tiles to go with a small margin of error. Go that distance in a couple directions from wherever you took the sample and it'll be there. triangulation's nice and all but it's pretty uncommon to have more than one digsite at the given distance and even if it is it's a quick clear so you can just ignore triangulation. You shouldn't be digging without a diamond drill so it's not a problem.
No need to give a single shit about the hyperspectral, do one artifact from one sample at a time If you aren't sampling the entire east side of the roid then processing them all you don't need to make sure you aren't going after the same artifact cause you already dug it out and it's no longer the nearest large when you take the next sample. I doubt we're ever going to get a home/tech digsite overhaul and have a xenoarch meme map where most of the server is running dig crews and analysis teams.

**Comment by Probe1** [comment]
Use trigonometry as the machines intend you to or return and rewrite xenoarch code for us
